Только в /root/dk_scripts/tmp/2: 1-Jul-2016131045LINK-W-Downfa9KTCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/2: 1-May-2014185454LINK-I-Upfa25KTCS4-SW1-48_192.168.22.71.conf
diff -r /root/dk_scripts/tmp/2/TCS0-SW1-24_192.168.22.62.conf /root/dk_scripts/tmp/1/TCS0-SW1-24_192.168.22.62.conf
161c161
<  switchport trunk allowed vlan add 2 
---
>  switchport trunk native vlan 66 
Только в /root/dk_scripts/tmp/1: TCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/1: TCS4-SW1-48_192.168.22.71.conf
